isaric/parsers/isaric_washington.toml

```diff
@@ -0,0 +1,614 @@
+
+[subject.works_microbiology_lab]
+field = "employed_as_a_healthcare_w"
+description = "Patient Employed as a Healthcare Worker or in a Microbiology Laboratory?"
+ref = "Y/N/NK"
+
+[subject.works_healthcare]
+field = "employed_as_a_healthcare_w"
+description = "Patient Employed as a Healthcare Worker or in a Microbiology Laboratory?"
+ref = "Y/N/NK"
+
+[subject.has_tuberculosis]
+field = "any_other_pos___4"
+ref = "Y/N/NK"
+
+[subject.has_dementia]
+field = "comorbidities___34"
+ref = "Y/N/NK"
+
+[subject.has_obesity]
+field = "comorbidities___30"
+ref = "Y/N/NK"
+
+[subject.has_hiv]
+combinedType = "any"
+fields = [
+    { field = "comorbidities___22", ref = "Y/N/NK" },
+    { field = "any_other_pos___1", ref = "Y/N/NK" }
+]
+
+[subject.has_hypertension]
+field = "comorbidities___3"
+ref = "Y/N/NK"
+
+[subject.has_malignant_neoplasm]
+field = "comorbidities___20"
+ref = "Y/N/NK"
+
+[subject.has_smoking]
+field = "social_history"
+values = { 1 = "Current Smoler", 2 = "Former Smoker", 3 = "Vaping", 4 = "Alcohol use disorder", 5 = "Substance use disorder", 0 = "Unknown/Not available" }
+
+[subject.has_asthma]
+field = "comorbidities___8"
+ref = "Y/N/NK"
+
+[subject.has_chronic_kidney_disease]
+field = "comorbidities___10"
+ref = "Y/N/NK"
+
+[subject.has_diabetes]
+field = "comorbidities___12"
+ref = "Y/N/NK"
+
+[subject.diabetes_type]
+field = "type_of_diabetes"
+
+[subject.diabetes_type.values]
+1 = "type-1"
+2 = "type-2"
+3 = "gestational"
+
+[subject.has_liver_disease]
+field = "comorbidities___14"
+ref = "Y/N/NK"
+
+[subject.has_malnutrition]
+field = "comorbidities___29"
+ref = "Y/N/NK"
+
+[subject.has_solid_organ_transplant]
+field = "comorbidities___21"
+ref = "Y/N/NK"
+
+[subject.has_hiv_art]
+field = "pre_hospital_medication___3"
+ref = "Y/N/NK"
+
+[subject.has_died] # ignores the person who died after discharge!  
+field = "hosp_status"
+description = "Hospital discharge status?"
+values = { 1 = true, 0 = false }
+
+[subject.icu_admitted]
+field = "admitted_to_icu"
+ref = "Y/N/NK"
+
+[visit]
+
+[visit.visit_id]
+field = "patient_study_id"
+sensitive = true
+
+[visit.subject_id]
+field = "patient_study_id"
+sensitive = true
+
+[visit.start_date] # fill here!
+ref = "admission_date"
+
+[visit.icu_admission]
+combinedType = "any"
+fields = [
+    { field = "admitted_to_icu", ref = "Y/N/NK" },
+    { field = "is_patient_admitted_to_icu", ref = "Y/N/NK" }
+]
+
+[visit.icu_admission_dates] # calculated from admission date + hospital day - CHECK IF THIS IS CORRECT?
+field = "admission_date"
+description = "icu admission date calculated from admission and hospital day"
+apply = { function = "endDate", params = ["day_to_icu"]}
```

Comment:
This is correct, but because ADMIT DATE and day_to_icu are on different lines in the data file, an error is thrown because one of the fields is always empty when endDate is run. We need to consider how to do this.

Reply:
sure ..